http/server.go

```diff
@@ -26,11 +28,13 @@ func (a *AppServer) initialize() {
 	privateChain := Chain(protected, Logger(a.Logger), CORS, NewAuthMiddleware(a.UserService))
 	publicChain := Chain(public, Logger(a.Logger), CORS)
 	staticChain := Chain(http.FileServer(http.Dir(a.Static)), Logger(a.Logger), CORS)
+	filesChain := Chain(http.StripPrefix("/files", http.FileServer(http.Dir(a.Files))), Logger(a.Logger), CORS)
```

Comment:
Shouldn't it be `Chain(http.FileServer(http.Dir(a.Files))), http.StripPrefix("/files")...`? The first argument of `Chain` is the final handler.